vta/include/vta/dpi/module.h

```diff
@@ -26,6 +26,10 @@
 #include <condition_variable>
 #include <string>
 
+#ifndef VTA_TSIM_USE_VIRTUAL_MEMORY
+#define VTA_TSIM_USE_VIRTUAL_MEMORY (1)
```

Comment:
are we going to leave this parameter turned on by default? what are the pros/cons to leaving this macro to be user-defined (with the idea that we're trying to minimize ifdefs in our code)

Reply:
#### Advantage
 * Enable tsim to test VTA subsystems that are configured to use with 32-bit memory address (checkout @vegaluisjose 's illustration at #3380)
#### Disadvantage
 * Current virtual memory system uses virtual memory page file system to write and read address mappings, these may not be compatible to Windows.

Otherwise, I think it's safe to enable virtual memory for `tsim_driver` by default.

Reply:
Thanks for the clear explanation. I'd opt towards leaving it on by default, at the detriment to Windows users, but others can chime in @tqchen @vegaluisjose